referrer-policy/spec.src.json

```diff
@@ -0,0 +1,293 @@
+      "referrer_policy": "no-referrer",
+      "test_expansion": [
+        {
+          "name": "generic",
+          "source_protocol": "*",
+          "target_protocol": "*",
+          "delivery_method": "*",
+          "origin": "*",
+          "subresource": "*",
+          "referrer_url": "omitted"
+        }
+      ]
+    },
+    {
+      "name": "no-referrer-when-downgrade",
+      "title": "Referrer Policy is set to 'no-referrer-when-downgrade'",
+      "description": "Check that non a priori insecure subresource gets the full Referrer URL. A priori insecure subresource gets no referrer information.",
+      "specification_url": "https://w3c.github.io/webappsec/specs/referrer-policy/#referrer-policy-state-no-referrer-when-downgrade",
+      "referrer_policy": "no-referrer-when-downgrade",
+      "test_expansion": [
+        {
+          "name": "insecure-protocol",
+          "source_protocol": "http",
+          "target_protocol": "http",
+          "delivery_method": ["http-csp", "meta-referrer", "meta-csp", "attr-referrer"],
+          "origin": "*",
+          "subresource": "*",
+          "referrer_url": "stripped-referrer"
+        },
+        {
+          "name": "upgrade-protocol",
+          "source_protocol": "http",
+          "target_protocol": "https",
+          "delivery_method": ["http-csp", "meta-referrer", "meta-csp", "attr-referrer"],
+          "origin": "*",
+          "subresource": "*",
+          "referrer_url": "stripped-referrer"
+        },
+        {
+          "name": "downgrade-protocol",
+          "source_protocol": "https",
+          "target_protocol": "http",
+          "delivery_method": ["http-csp", "meta-referrer", "meta-csp", "attr-referrer"],
+          "origin": "*",
+          "subresource": "*",
+          "referrer_url": "origin"
+        },
+        {
+          "name": "secure-protocol",
+          "source_protocol": "https",
+          "target_protocol": "https",
+          "delivery_method": ["http-csp", "meta-referrer", "meta-csp", "attr-referrer"],
+          "origin": "*",
+          "subresource": "*",
+          "referrer_url": "stripped-referrer"
+        }
+      ]
+    },
+    {
+      "name": "origin-only",
+      "title": "Referrer Policy is set to 'origin-only'",
+      "description": "Check that all subresources in all casses get only the origin portion of the referrer URL.",
+      "specification_url": "https://w3c.github.io/webappsec/specs/referrer-policy/#referrer-policy-state-origin",
+      "referrer_policy": "origin",
+      "test_expansion": [
+        {
+          "name": "generic",
+          "source_protocol": "*",
+          "target_protocol": "*",
+          "delivery_method": ["http-csp", "meta-referrer", "meta-csp", "attr-referrer"],
+          "origin": "*",
+          "subresource": "*",
+          "referrer_url": "origin"
+        }
+      ]
+    },
+    {
+      "name": "origin-when-crossorigin",
```

Comment:
Yes, I've seen that CL and am aware of the issue regarding these tests as well.
I'll add this to my personal TODO list. 
I'll also see what I can do to allow adding comments to the spec JSON.